src/components/profilepaymentmethods.main.jsx

```diff
@@ -74,7 +74,7 @@ class ProfilePaymentMethodsMain extends React.Component {
                     {displayName}
                   </span>
                 </div>
-                <button className="btn profile-delete-payment-btn" type="button" onClick={() => { this.handleDelete(paymentElement.self.uri); }}>
+                <button className="vestri-btn small" type="button" onClick={() => { this.handleDelete(paymentElement.self.uri); }}>
```

Comment:
The independent class names (example: profile-delete-payment-btn) are meant more for readability and for automation. I'd prefer to keep some or add IDs where applicable rather than having generic elements in some places.